src/liquid/components/ld-bg-cells/ld-bg-cells.tsx

```diff
@@ -0,0 +1,57 @@
+import { Component, Element, getAssetPath, h, Host, Prop } from '@stencil/core'
+import { getClassNames } from '../../utils/getClassNames'
+import '../../components' // type definitions for type checks and intelliSense
+
+type CellType =
+  | 'bioreliance'
+  | 'f'
+  | 'hexagon'
+  | 'millipore'
+  | 'qa-x2f-qc'
+  | 'safc'
+  | 'sigma-aldrich'
+  | 't'
+  | 'tile'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  assetsDirs: ['assets'],
+  tag: 'ld-bg-cells',
+  styleUrl: 'ld-bg-cells.css',
+  shadow: false,
+})
+export class LdBgCells {
+  @Element() element: HTMLElement
```

Comment:
I think you are not using the element anywhere, so you should be safe to remove it.